includes/class-sensei-course.php

```diff
@@ -3072,7 +3081,13 @@ public static function output_course_enrolment_actions() {
 
 				}
 
-				if ( ! (bool) apply_filters( 'sensei_user_can_register_for_course', true, $post->ID ) ) {
+				if (
+					! (bool) apply_filters_deprecated(
```

Comment:
Here's what this looks like in the logs:
```
PHP Notice:  sensei_couse_access_permission_message is <strong>deprecated</strong> since version 3.0.0! Use  instead.
```

Seems like a WP bug, but I still wonder if it wouldn't be better to pass `null` as the fourth parameter, which results in a much better PHP notice:
```
PHP Notice:  sensei_couse_access_permission_message is <strong>deprecated</strong> since version 3.0.0 with no alternative available.
```

Reply:
Good catch, Donna! Fixed here: https://github.com/Automattic/sensei/pull/2979/commits/399eb26cfe6e9e3002460a27a8160232ba5de3b6

And I opened a ticket here: https://core.trac.wordpress.org/ticket/49698#ticket